__tests__/nc_news_db.test.js

```diff
@@ -31,7 +31,54 @@ describe("GET /api/", () => {
 			.get("/api")
 			.expect(200)
 			.then(({ body }) => {
-				expect(body).toEqual(endpoints);
+				expect(body.endpoints).toEqual(endpoints);
+			});
+	});
+});
+
+describe("GET /api/articles", () => {
+	it("status: 200, should respond with all articles and their properties", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
+			.then(({ body }) => {
+				const articlesArray = body.articles;
+				articlesArray.forEach((article) => {
```

Comment:
check the length of the array before your forEach so you know its not a false positive